default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, buildGoModule, fetchFromGitHub, installShellFiles }:
 buildGoModule {
-  pname   = "oplogtoredis";
-  version = "2.0.1";
+  pname = "oplogtoredis";
+  version = "3.0.0";
 
   src = builtins.path { path = ./.; };
 
   vendorSha256 = "sha256-VHiYVJUNtHN2IY4iXZ6kHAa3Avi2VwRH1ySKBrrCDu4=";
-  postInstall  = ''
+  postInstall = ''
   '';
-  nativeBuildInputs = [installShellFiles];
-  doCheck           = false;
-  doInstallCheck    = false;
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
Is there a nix linter?

Reply:
Not that I know of.

Reply:
https://discourse.nixos.org/t/list-of-nix-linters/19279